src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ApiCompatServiceProvider.cs

```diff
@@ -27,21 +27,12 @@ public ApiCompatServiceProvider(Func<ISuppressionEngine, ISuppressibleLog> logFa
             _compatibilityLogger = new Lazy<ISuppressibleLog>(() => logFactory(SuppressionEngine));
             _apiCompatRunner = new Lazy<IApiCompatRunner>(() =>
             {
-                AccessibilitySymbolFilter accessibilitySymbolFilter = new(respectInternals);
                 SymbolEqualityComparer symbolEqualityComparer = new();
 
-                // The attribute data symbol filter is a composite that contains both the accessibility
-                // symbol filter and the doc id symbol filter.
-                CompositeSymbolFilter attributeDataSymbolFilter = new(accessibilitySymbolFilter);
-                if (excludeAttributesFiles is not null)
-                {
-                    attributeDataSymbolFilter.Add(new DocIdSymbolFilter(excludeAttributesFiles));
-                }
-
                 ApiComparerSettings apiComparerSettings = new(
-                    accessibilitySymbolFilter,
+                    symbolFilter: null, // Gets a default value inside the constructor
```

Comment:
This line looks a bit questionable.

Reply:
Why? See the behavior inside

Reply:
https://github.com/dotnet/sdk/blob/e1767073596e007b503ca6324957e13c77b0c045/src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompatibility/ApiComparerSettings.cs#L46

Reply:
consider just omitting the parameter and using named parameters for all if you think it's more readable.  The null is a little odd.

Reply:
Yeah I can do that, but the comment I added next to the null had the purpose of informing the reason.

I guess if I omit this parameter we don't have to maintain the comment. I'll do that.